pkgs/applications/editors/eclipse/plugins.nix

```diff
@@ -108,6 +108,7 @@ rec {
     meta = with lib; {
       homepage = "https://github.com/tobiasmelcher/EclipseAceJump";
       description = "Provides fast jumps to text based on initial letter";
+      sourceProvenance = with sourceTypes; [ binaryBytecode ];
```

Comment:
I would suggest to move this to buildEclipsePlugin since probably all or most plugins are like this.

Reply:
I think some people might find that to be surprising behaviour.